BedrockCore.cpp

```diff
@@ -159,7 +159,7 @@ BedrockCore::RESULT BedrockCore::peekCommand(unique_ptr<BedrockCommand>& command
 }
 
 BedrockCore::RESULT BedrockCore::processCommand(unique_ptr<BedrockCommand>& command, bool exclusive) {
-    AutoTimer timer(command, BedrockCommand::PROCESS);
+    AutoTimer timer(command, BedrockCommand::PROCESS, exclusive ? BedrockCommand::BLOCKING_PROCESS : BedrockCommand::NONE);
```

Comment:
This doesn't make any sense to me? Why don't we just use the existing `timer` constructor and pass:
```
    AutoTimer timer(command, exclusive ? BedrockCommand::BLOCKING_PROCESS : BedrockCommand::PROCESS);
```

Reply:
We could either add them together or just log them separately as "blocking" and "non-blocking" process. I don't care that much about how the log line looks so long as it's clear. I think it makes more sense to count them separately with the timer though, we don't need the extra parameter.

Reply:
>so we keep the old metric the same

Agreed with keeping the old metric the same